include/api/types.h

```diff
@@ -135,8 +135,8 @@ extern struct debug_syscall_error current_debug_error;
  */
 #define userError(M, ...) \
     do {                                                                       \
-        out_error(ANSI_DARK "<<" ANSI_GREEN "seL4(CPU %" SEL4_PRIu_word ")"    \
-                ANSI_DARK " [%s/%d T%p \"%s\" @%lx]: " M ">>" ANSI_RESET "\n", \
+        out_error(ANSI_BOLD "<<" ANSI_GREEN "seL4(CPU %" SEL4_PRIu_word ")"    \
```

Comment:
Nice PR, completely unrelated question for whoever to answer: in addition to printing the CPU would it be a good idea to print the current domain? `NODE_STATE(ksCurThread)->tcbDomain` this can be helpful to quickly determine which domain the error is coming from.